src/main/java/seedu/address/logic/Messages.java

```diff
@@ -18,7 +18,9 @@ public class Messages {
     public static final String MESSAGE_EMPTY_FIND_KEYWORD = "Find keyword(s) cannot be empty!";
     public static final String MESSAGE_FIND_KEYWORD_CONTAINS_WHITESPACE =
             "Find keyword(s) cannot contain whitespace(s)!";
+    public static final String MESSAGE_MISSING_INDEX = "Error: Missing index for command! \nUsage:\n%1$s";
     public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid";
+    public static final String MESSAGE_PERSON_INDEX_OUT_OF_BOUND = "Error: The person index provided is out of bound";
```

Comment:
```suggestion
    public static final String MESSAGE_PERSON_INDEX_OUT_OF_BOUND = "Error: The person index provided is out of bounds";
```

`out of bound` -> `out of bounds`